gulpfile.js

```diff
@@ -115,6 +96,22 @@ gulp.task('sass:build', ['webfont'], function() {
     .pipe(gulp.dest(config.folderDev.css));
 });
 
+gulp.task('sass:dist', ['webfont'], function() {
```

Comment:
This code is almost the same as _sass_ task, did you consider reuse the code?

I am not so familiar with gulp tasks, maybe it is better having clear and duplicated code than ugly reusings.

Reply:
I researched this, and is better this way for now. I did not find another way around.